From 087cf43450f11e651756fd99caf24f6cd9b02c78 Mon Sep 17 00:00:00 2001
From: Cristina Ciocan <cristina-mihaela.ciocan@nxp.com>
Date: Tue, 28 Feb 2017 19:10:54 +0200
Subject: [PATCH 1546/5242] MLK-14292: 4.9 rebase: LCDIF: 6SX SDB does not
 boot with lcdif dts

commit  3507b7a1511e23a4046ba7803194ad342cbb2a3d from
https://source.codeaurora.org/external/imx/linux-imx.git

On imx6SX SDB eLCDIF driver cannot coexist with CAAM driver. The cause
seems to be an ipg clk that failes to be identified by the CAAM driver.

This patch disables the CAAM driver by setting its status to "disabled"
in the dts used for obtaining LCDIF functionality.

Signed-off-by: Cristina Ciocan <cristina-mihaela.ciocan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts b/arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts
index 9391e75..48e242e 100644
--- a/arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts
@@ -37,3 +37,7 @@
 &ov5640 {
 	status = "disabled";
 };
+
+&crypto {
+	status = "disabled";
+};
-- 
1.7.9.5

